db/migrate/20180205113216_add_screener_answers.rb

```diff
@@ -0,0 +1,9 @@
+class AddScreenerAnswers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :screener_answers do |t|
```

Comment:
We use `uuids` for primary keys in all tables. Not that it really matters in this table as we are not going to expose the records publicly for CRUD or anything like that, but for consistency, it might be a good idea to continue using uuids. Just adding `id: :uuid` will do the trick. Otherwise it generates a numeric ID pk by default.

Also, maybe timestamp fields (`t.timestamps null: false`) could be interesting if we were to do internal analysis of records in this table, to order and group by date, and create reports 🤔